src/main/java/io/pravega/connectors/hadoop/Helper.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.connectors.hadoop;
+
+import com.google.common.base.Preconditions;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.Base64;
+import java.util.UUID;
+
+public class Helper {
```

Comment:
The name of the class is too general. I'd suggest qualifying the name with another word (for example, TruststoreHelper or a more general form if needed) that makes it clear what functionality the class is assisting with.

Reply:
renamed the file